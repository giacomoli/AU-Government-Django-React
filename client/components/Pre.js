import React, { Fragment } from 'react'

export default ({ children }) => (
  <Fragment>
    <pre>{children}</pre>
    <style jsx>{`
      pre {
        box-shadow: none;
        color: #67757f;
      }
    `}</style>
  </Fragment>
)
