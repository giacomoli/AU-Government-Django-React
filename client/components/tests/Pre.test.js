import React from 'react'
import renderer from 'react-test-renderer'
import Pre from '../Pre'

/* eslint-disable */
describe('Pre', () => {

  it('should render Pre without props', () => {
    const wrapper = renderer.create(
      <Pre />
    )
    expect(wrapper).toMatchSnapshot()
  })

  it('should render Pre with props', () => {
    const wrapper = renderer.create(
      <Pre children={"Test children"} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
