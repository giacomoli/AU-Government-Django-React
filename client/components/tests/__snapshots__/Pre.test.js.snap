// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pre should render Pre with props 1`] = `
<pre
  className="jsx-2704498154"
>
  Test children
</pre>
`;

exports[`Pre should render Pre without props 1`] = `
<pre
  className="jsx-2704498154"
/>
`;
